conf/application.conf

```diff
@@ -119,6 +124,11 @@ Test {
         start.path = "agent-services-account"
       }
 
+      government-gateway-authentication {
```

Comment:
dev is for local environment, this is how is defined from play framework. The Prod environment is the actual profile configuration that is being overwritten by app-config-*